workspace.dsl

```diff
@@ -0,0 +1,167 @@
+workspace "Appeal service" {
+	model {
+		properties {
+			"structurizr.groupSeparator" "/"
+		}
+
+		userLocalPlanningAuthority = person "Local Planning Authority" "A local authority issuing decisions on planning applications"
+		userAppellant = person "Appellant" "A member of the public appealing against a planning decision"
+
+		group "Planning Inspectorate" {
+			userCaseOfficer = person "Case officer" "Checks LPA questionnaire and uploads statements and comments"
+			userInspector = person "Inspector" "Visits appeal site and issues appeal decision"
+
+			systemAppsFo = softwareSystem "Appeals Front-Office" "Public dashboards for appellants and Local Planning Authorities" "Node.js, Azure Web App, Azure Function App" {
+				tags "InternalCollaborationSystem"
+
+				containerFoFileStorage = container "File storage" "Stores all files uploaded in the Front-Office"
+			}
+
+			systemAppsBo = softwareSystem "Appeals Back-Office" "Internally facing service allowing for management of planning appeals in England"  {
+				tags "InternalCollaborationSystem"
+
+				group "Web" {
+
+    				containerBoWeb = container "Back-Office Web" "Back-office website" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - Web"
+    				}
+
+    				containerBoRedis = container "Redis" "Distributed session storage"
+    				containerBoAzureActiveDirectory = container "Entra ID" "Stores login details and assigned user groups"
+    				containerBoFileStorage = container "File storage" "Stores all files related to appeals"
+
+				}
+
+				group "API" {
+    				containerBoApi = container "Back-Office API" "" "Node.js, Azure Web App" {
+    					tags "Microsoft Azure - App Services - API"
+    					componentBoStateMachine = component "State machine" "Handles the appeal lifecycle and state transitions"
+    				}
+
+    				containerBoAzureSql = container "Database" "Data store for appeals" "Azure SQL"
```

Comment:
If you give this a tag of `Database` then this will pickup the theme style and render as a cylinder, which is a nice touch